src/transforms/filter.jl

```diff
@@ -55,20 +56,36 @@ Drop all rows with missing values in selects columns `col₁`, `col₂`, ..., `c
 
 Drop all rows with missing values in columns that match with `regex`.
 """
-DropMissing() = Filter(row -> all(!ismissing, row))
+struct DropMissing{S<:ColSpec} <: Stateless
+  colspec::S
+end
 
-DropMissing(cols::VecOrTuple{T}) where {T<:Union{Symbol, Integer}} =
-  Filter(row -> all(!ismissing, getindex.(Ref(row), cols)))
+# argument error
+DropMissing(::Tuple{}) = throw(ArgumentError("Cannot create a DropMissing object with empty tuple."))
 
-DropMissing(cols::VecOrTuple{T}) where {T<:AbstractString} =
-  DropMissing(Symbol.(cols))
+DropMissing() = DropMissing(:)
 
 DropMissing(cols::T...) where {T<:ColSelector} =
   DropMissing(cols)
 
-function DropMissing(regex::Regex)
-  Filter() do row
-    cols = _filter(regex, propertynames(row))
-    all(!ismissing, getindex.(Ref(row), cols))
-  end
+isrevertible(::Type{<:DropMissing}) = true
+
+_ftrans(::DropMissing{Colon}, table) =
+  Filter(row -> all(!ismissing, row))
```

Comment:
Can we avoid introducing this helper function, and instead use the DropMissing constructor to return a Filter object?

Reply:
I don't think it makes sense to use the constructor of one type to return an object of another type.
But we can use Functors.

Reply:
Agree, the functor interface is already provided as a fallback to all Transform types. The helper function seems like the best alternative.